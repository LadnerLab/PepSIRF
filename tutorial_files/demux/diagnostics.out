Sample name	Index1 Matches	Index Pair Matches	DNA Tag Matches
P92_5.1.20_CovA.1F	156	9	9
P92_5.8.20_CovA.1G	156	7	5
Nor.27_1X_PCV_1x_ProG_A	156	10	6
Nor.28_1X_PCV_1x_ProG_A	156	8	7
Nor.29_1X_PCV_1x_ProG_A	156	13	8
Nor.30_1X_PCV_1x_ProG_A	156	7	6
Nor.31_1X_PCV_1x_ProG_A	156	14	9
Nor.32_1X_PCV_1x_ProG_A	156	12	9
Nor.33_1X_PCV_1x_ProG_A	156	14	11
Nor.34_1X_PCV_1x_ProG_A	156	5	2
P78_5.1.20_CovA.1H	156	4	4
P60_5.1.20_CovA.1A	156	11	10
P62_5.1.20_CovA.1B	156	7	4
P62_5.5.20_CovA.1C	156	12	11
P58_5.1.20_CovA.1D	156	10	10
P58_5.8.20_CovA.1E	156	4	3
D10_4.2.20_8.8H	194	5	3
D06_5.1.20_8.9F	194	5	4
BB.8_1X_PCV_1x_ProG_B	194	22	19
BB.9_1X_PCV_1x_ProG_B	194	12	11
BB.10_1X_PCV_1x_ProG_B	194	17	10
BB.11_1X_PCV_1x_ProG_B	194	17	11
BB.12_1X_PCV_1x_ProG_B	194	12	8
BB.13_1X_PCV_1x_ProG_B	194	18	10
BB.14_1X_PCV_1x_ProG_B	194	8	3
JA.s_1X_PCV_1X_ProG_E	194	12	10
D08_5.1.2.20_8.9G	194	5	5
P03_2.25.20_8.2A	194	18	14
P10_2.21.20_8.2D	194	9	7
D09_3.8.20_8.5B	194	3	2
D08_3.18.20_8.5C	194	5	5
D06_3.19.20_8.5D	194	19	16
D15_5.1.20_8.10H	178	18	13
D16_5.1.20_8.11A	178	2	2
D09_5.11.20_8.11C	178	5	5
D11_4.28.20_8.10C	177	12	11
D10_5.6.20_8.10D	178	6	4
D13_5.1.20_8.10E	178	10	8
D12_5.1.20_8.10F	178	5	4
D14_5.1.20_8.10G	178	10	10
D18_5.31.20_8.12E	238	13	11
D19_5.24.20_8.12F	238	5	5
D19_5.31.20_8.12G	239	6	4
P10_5.19.20_8.11D	239	31	25
P03_5.18.20_8.11E	239	21	19
D17_5.24.20_8.12B	237	8	7
D17_5.31.20_8.12C	238	3	3
D18_5.25.20_8.12D	238	10	10
P25_2.26.20_8.4F	428	0	0
P10_3.16.20_8.4G	428	19	18
BB.15_1X_PCV_1x_ProG_A	428	6	1
BB.16_1X_PCV_1x_ProG_A	428	5	2
BB.17_1X_PCV_1x_ProG_A	428	10	6
BB.18_1X_PCV_1x_ProG_A	428	7	3
BB.19_1X_PCV_1x_ProG_A	428	1	1
BB.20_1X_PCV_1x_ProG_A	427	15	5
BB.1_1X_PCV_1x_ProG_A	428	3	0
BB.2_1X_PCV_1x_ProG_A	426	1	0
P18_3.15.20_8.4H	428	10	10
P40_11.2.2019_8.4A	428	75	62
P40_11.26.2019_8.4B	428	63	54
P40_1.31.2019_8.4C	427	93	84
P40_2.25.20_8.4D	427	37	33
P24_3.2.20_8.4E	427	75	62
Superblock_PCV_D	393	54	51
JA_Serum_PCV_C	392	8	8
BB.15_1X_PCV_1x_ProG_B	393	11	8
BB.16_1X_PCV_1x_ProG_B	394	3	3
BB.17_1X_PCV_1x_ProG_B	394	4	2
BB.18_1X_PCV_1x_ProG_B	394	3	0
BB.19_1X_PCV_1x_ProG_B	394	0	0
BB.20_1X_PCV_1x_ProG_B	394	9	5
BB.1_1X_PCV_1x_ProG_B	394	3	0
BB.2_1X_PCV_1x_ProG_B	394	3	0
JA_Serum_PCV_D	394	13	13
P11_1.27.20_8.1E	394	72	64
P24_1.31.20_8.1F	394	40	33
P25_1.27.20_8.1G	394	78	69
P18_2.21.20_8.1H	394	23	19
Superblock_PCV_C	394	60	56
Nor.43_1X_PCV_1x_ProG_A	59	9	7
Nor.44_1X_PCV_1x_ProG_A	59	2	2
HCV.Q_1X_PCV_1x_ProG_A	59	5	4
V1_1X_PCV_1x_ProG_A	59	3	2
V2_1X_PCV_1x_ProG_A	59	11	9
V4_1X_PCV_1x_ProG_A	59	11	8
V7_1X_PCV_1x_ProG_A	59	7	3
V8_1X_PCV_1x_ProG_A	59	7	2
Nor.43_1X_PCV_1x_ProG_B	52	5	4
Nor.44_1X_PCV_1x_ProG_B	52	8	4
HCV.Q_1X_PCV_1x_ProG_B	52	11	5
V1_1X_PCV_1x_ProG_B	52	4	4
V2_1X_PCV_1x_ProG_B	52	2	1
V4_1X_PCV_1x_ProG_B	52	13	6
V7_1X_PCV_1x_ProG_B	52	3	1
V8_1X_PCV_1x_ProG_B	52	3	3
P1_1X_PCV_1x_ProG_A	69	5	4
P2_1X_PCV_1x_ProG_A	69	5	2
P3_1X_PCV_1x_ProG_A	69	6	4
P4_1X_PCV_1x_ProG_A	69	10	8
HD6_1X_PCV_1x_ProG_A	69	2	2
P56_5.1.20_CovA.2F	294	7	4
P56_5.8.20_CovA.2G	294	3	3
Nor.27_1X_PCV_1x_ProG_B	294	24	19
Nor.28_1X_PCV_1x_ProG_B	294	16	10
Nor.29_1X_PCV_1x_ProG_B	294	21	15
Nor.30_1X_PCV_1x_ProG_B	294	6	3
Nor.31_1X_PCV_1x_ProG_B	294	3	1
Nor.32_1X_PCV_1x_ProG_B	294	14	10
Nor.33_1X_PCV_1x_ProG_B	294	15	6
Nor.34_1X_PCV_1x_ProG_B	294	14	7
P69_5.2.20_CovA.2H	294	2	2
P78_5.9.20_CovA.2A	294	56	52
P43_5.1.20_CovA.2B	294	42	37
P43_5.7.20_CovA.2C	294	31	27
P52_4.30.20_CovA.2D	294	16	16
P52_5.7.20_CovA.2E	294	16	14
P1_1X_PCV_1x_ProG_B	55	5	4
P2_1X_PCV_1x_ProG_B	55	0	0
P3_1X_PCV_1x_ProG_B	55	12	9
P4_1X_PCV_1x_ProG_B	55	8	7
HD6_1X_PCV_1x_ProG_B	55	6	5
HD7_1X_PCV_1x_ProG_A	75	21	16
HD8_1X_PCV_1x_ProG_A	75	9	3
HD9_1X_PCV_1x_ProG_A	75	12	9
BB3_1X_PCV_1x_ProG_A	75	6	3
V3_1X_PCV_1x_ProG_A	75	9	9
V5_1X_PCV_1x_ProG_A	75	15	9
V6_1X_PCV_1x_ProG_A	75	1	0
P5_1X_PCV_1x_ProG_A	75	1	0
HD7_1X_PCV_1x_ProG_B	56	9	5
HD8_1X_PCV_1x_ProG_B	56	9	6
HD9_1X_PCV_1x_ProG_B	56	9	4
BB3_1X_PCV_1x_ProG_B	56	6	4
V3_1X_PCV_1x_ProG_B	56	1	1
V5_1X_PCV_1x_ProG_B	56	11	8
V6_1X_PCV_1x_ProG_B	56	5	4
P5_1X_PCV_1x_ProG_B	56	2	2
BB4_1X_PCV_1x_ProG_A	73	12	6
BB5_1X_PCV_1x_ProG_A	73	20	12
BB6_1X_PCV_1x_ProG_A	73	13	7
BB7_1X_PCV_1x_ProG_A	73	3	0
BB4_1X_PCV_1x_ProG_B	60	5	3
BB5_1X_PCV_1x_ProG_B	60	11	7
BB6_1X_PCV_1x_ProG_B	59	10	8
BB7_1X_PCV_1x_ProG_B	60	0	0
Sblk_1X_PCV_1X_ProG_F	59	15	15
JA.s_1X_PCV_1X_ProG_F	59	8	7
P77_5.4.20_CovA.3F	225	14	11
P77_5.11.20_CovA.3G	227	3	3
P88_5.5.20_CovA.3H	227	3	2
P69_5.9.20_CovA.3A	227	30	25
P81_5.2.20_CovA.3B	227	24	22
P81_5.9.20_CovA.3C	227	15	13
P51_5.3.20_CovA.3D	227	6	5
P51_5.10.20_CovA.3E	227	4	3
P87_5.6.20_CovA.4F	219	7	7
P87_5.13.20_CovA.4G	219	2	2
P73_5.4.20_CovA.4H	218	1	1
P88_5.12.20_CovA.4A	218	34	31
P90_5.6.20_CovA.4B	219	26	23
P90_5.13.20_CovA.4C	219	16	15
P48_5.3.20_CovA.4D	219	13	11
P48_5.10.20_CovA.4E	217	13	10
P61_5.14.20_CovA.5F	208	15	15
P44_5.9.20_CovA.5G	208	1	0
Nor.35_1X_PCV_1x_ProG_A	207	9	8
Nor.36_1X_PCV_1x_ProG_A	207	21	13
Nor.37_1X_PCV_1x_ProG_A	208	12	8
Nor.38_1X_PCV_1x_ProG_A	208	11	10
Nor.39_1X_PCV_1x_ProG_A	207	7	6
Nor.40_1X_PCV_1x_ProG_A	208	19	13
Nor.41_1X_PCV_1x_ProG_A	208	12	12
Nor.42_1X_PCV_1x_ProG_A	208	12	8
P44_Unk_CovA.5H	208	1	1
P73_5.11.20_CovA.5A	208	24	22
P82_5.13.20_CovA.5B	208	26	24
P47_5.8.20_CovA.5C	208	11	11
P47_5.15.20_CovA.5D	208	11	11
P61_5.7.20_CovA.5E	208	8	7
P83_5.9.20_CovA.6F	270	11	11
P83_5.16.20_CovA.6G	271	3	2
Nor.35_1X_PCV_1x_ProG_B	270	21	16
Nor.36_1X_PCV_1x_ProG_B	270	18	15
Nor.37_1X_PCV_1x_ProG_B	270	14	13
Nor.38_1X_PCV_1x_ProG_B	270	15	12
Nor.39_1X_PCV_1x_ProG_B	270	7	6
Nor.40_1X_PCV_1x_ProG_B	270	16	12
Nor.41_1X_PCV_1x_ProG_B	270	14	12
Nor.42_1X_PCV_1x_ProG_B	270	22	13
P55_5.9.20_CovA.6H	270	4	4
P57_5.13.20_CovA.6A	270	41	36
P46_5.9.20_CovA.6B	270	28	24
P46_5.17.20_CovA.6C	270	12	10
P64_5.8.20_CovA.6D	270	11	10
P64_5.15.20_CovA.6E	270	22	20
P53_5.16.20_CovA.7F	218	14	12
P86_5.14.20_CovA.7G	218	2	2
P86_5.20.20_CovA.7H	216	1	1
P55_5.16.20_CovA.7A	216	18	15
P65_5.11.20_CovA.7B	219	16	15
P65_5.18.20_CovA.7C	220	7	7
P59_Unk_CovA.7D	220	7	6
P53_5.9.20_CovA.7E	218	14	14
P93_5.29.20_CovA.8F	225	19	17
P91_5.21.20_CovA.8G	225	7	7
P91_5.28.20_CovA.8H	225	6	6
P63_5.16.20_CovA.8A	225	15	14
P63_5.23.20_CovA.8B	225	10	8
P84_5.15.20_CovA.8C	225	19	17
P84_5.22.20_CovA.8D	225	14	14
P93_5.19.20_CovA.8E	225	11	11
Superblock_PCV_B	200	1	0
JA_Serum_PCV_A	200	0	0
BB.8_1X_PCV_1x_ProG_A	200	17	13
BB.9_1X_PCV_1x_ProG_A	200	17	16
BB.10_1X_PCV_1x_ProG_A	200	7	4
BB.11_1X_PCV_1x_ProG_A	200	11	9
BB.12_1X_PCV_1x_ProG_A	200	10	7
BB.13_1X_PCV_1x_ProG_A	200	13	10
BB.14_1X_PCV_1x_ProG_A	199	11	5
Sblk_1X_PCV_1X_ProG_E	200	44	41
JA_Serum_PCV_B	200	1	0
P74_5.11.20_CovA.9A	200	24	22
P74_5.18.20_CovA.9B	199	15	14
P45_5.16.20_CovA.9C	200	12	10
P45_5.25.20_CovA.9D	200	8	7
Superblock_PCV_A	200	2	0

