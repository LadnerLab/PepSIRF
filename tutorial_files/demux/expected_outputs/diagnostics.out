Sample name	Index1 Matches	Index Pair Matches	DNA Tag Matches
P92_5.1.20_CovA.1F	156	9	9
P92_5.8.20_CovA.1G	156	7	5
Nor.27_1X_PCV_1x_ProG_A	155	10	6
Nor.28_1X_PCV_1x_ProG_A	155	8	7
Nor.29_1X_PCV_1x_ProG_A	155	13	8
Nor.30_1X_PCV_1x_ProG_A	155	7	6
Nor.31_1X_PCV_1x_ProG_A	155	14	9
Nor.32_1X_PCV_1x_ProG_A	155	12	9
Nor.33_1X_PCV_1x_ProG_A	155	14	11
Nor.34_1X_PCV_1x_ProG_A	155	5	2
P78_5.1.20_CovA.1H	156	4	4
P60_5.1.20_CovA.1A	156	11	10
P62_5.1.20_CovA.1B	156	7	4
P62_5.5.20_CovA.1C	155	12	11
P58_5.1.20_CovA.1D	155	10	10
P58_5.8.20_CovA.1E	155	4	3
D10_4.2.20_8.8H	194	5	3
D06_5.1.20_8.9F	194	5	4
BB.8_1X_PCV_1x_ProG_B	193	22	19
BB.9_1X_PCV_1x_ProG_B	193	12	11
BB.10_1X_PCV_1x_ProG_B	193	17	10
BB.11_1X_PCV_1x_ProG_B	193	17	11
BB.12_1X_PCV_1x_ProG_B	193	12	8
BB.13_1X_PCV_1x_ProG_B	193	18	10
BB.14_1X_PCV_1x_ProG_B	193	8	3
JA.s_1X_PCV_1X_ProG_E	193	12	10
D08_5.1.2.20_8.9G	192	5	5
P03_2.25.20_8.2A	192	18	14
P10_2.21.20_8.2D	192	9	7
D09_3.8.20_8.5B	192	3	2
D08_3.18.20_8.5C	192	5	5
D06_3.19.20_8.5D	193	19	16
D15_5.1.20_8.10H	178	18	13
D16_5.1.20_8.11A	178	2	2
D09_5.11.20_8.11C	177	5	5
D11_4.28.20_8.10C	177	12	11
D10_5.6.20_8.10D	177	6	4
D13_5.1.20_8.10E	177	10	8
D12_5.1.20_8.10F	177	5	4
D14_5.1.20_8.10G	177	10	10
D18_5.31.20_8.12E	239	13	11
D19_5.24.20_8.12F	239	5	5
D19_5.31.20_8.12G	239	6	4
P10_5.19.20_8.11D	239	31	25
P03_5.18.20_8.11E	238	21	19
D17_5.24.20_8.12B	239	8	7
D17_5.31.20_8.12C	238	3	3
D18_5.25.20_8.12D	238	10	10
P25_2.26.20_8.4F	421	0	0
P10_3.16.20_8.4G	424	19	18
BB.15_1X_PCV_1x_ProG_A	423	6	1
BB.16_1X_PCV_1x_ProG_A	424	5	2
BB.17_1X_PCV_1x_ProG_A	423	10	6
BB.18_1X_PCV_1x_ProG_A	425	7	3
BB.19_1X_PCV_1x_ProG_A	425	1	1
BB.20_1X_PCV_1x_ProG_A	424	15	5
BB.1_1X_PCV_1x_ProG_A	424	3	0
BB.2_1X_PCV_1x_ProG_A	424	1	0
P18_3.15.20_8.4H	420	10	10
P40_11.2.2019_8.4A	418	75	62
P40_11.26.2019_8.4B	419	63	54
P40_1.31.2019_8.4C	420	94	84
P40_2.25.20_8.4D	419	37	33
P24_3.2.20_8.4E	420	75	62
Superblock_PCV_D	394	54	51
JA_Serum_PCV_C	392	8	8
BB.15_1X_PCV_1x_ProG_B	394	11	8
BB.16_1X_PCV_1x_ProG_B	394	3	3
BB.17_1X_PCV_1x_ProG_B	393	4	2
BB.18_1X_PCV_1x_ProG_B	394	3	0
BB.19_1X_PCV_1x_ProG_B	394	0	0
BB.20_1X_PCV_1x_ProG_B	392	9	5
BB.1_1X_PCV_1x_ProG_B	394	3	0
BB.2_1X_PCV_1x_ProG_B	394	3	0
JA_Serum_PCV_D	393	13	13
P11_1.27.20_8.1E	393	72	64
P24_1.31.20_8.1F	393	40	33
P25_1.27.20_8.1G	393	78	69
P18_2.21.20_8.1H	394	23	19
Superblock_PCV_C	393	60	56
Nor.43_1X_PCV_1x_ProG_A	58	9	7
Nor.44_1X_PCV_1x_ProG_A	58	2	2
HCV.Q_1X_PCV_1x_ProG_A	58	5	4
V1_1X_PCV_1x_ProG_A	58	3	2
V2_1X_PCV_1x_ProG_A	58	11	9
V4_1X_PCV_1x_ProG_A	58	11	8
V7_1X_PCV_1x_ProG_A	58	7	3
V8_1X_PCV_1x_ProG_A	58	7	2
Nor.43_1X_PCV_1x_ProG_B	52	5	4
Nor.44_1X_PCV_1x_ProG_B	52	8	4
HCV.Q_1X_PCV_1x_ProG_B	52	11	5
V1_1X_PCV_1x_ProG_B	52	4	4
V2_1X_PCV_1x_ProG_B	52	2	1
V4_1X_PCV_1x_ProG_B	52	13	6
V7_1X_PCV_1x_ProG_B	52	3	1
V8_1X_PCV_1x_ProG_B	52	3	3
P1_1X_PCV_1x_ProG_A	69	5	4
P2_1X_PCV_1x_ProG_A	69	5	2
P3_1X_PCV_1x_ProG_A	69	6	4
P4_1X_PCV_1x_ProG_A	68	10	8
HD6_1X_PCV_1x_ProG_A	68	2	2
P56_5.1.20_CovA.2F	292	7	4
P56_5.8.20_CovA.2G	292	3	3
Nor.27_1X_PCV_1x_ProG_B	292	24	19
Nor.28_1X_PCV_1x_ProG_B	292	16	10
Nor.29_1X_PCV_1x_ProG_B	293	21	15
Nor.30_1X_PCV_1x_ProG_B	293	6	3
Nor.31_1X_PCV_1x_ProG_B	292	3	1
Nor.32_1X_PCV_1x_ProG_B	294	14	10
Nor.33_1X_PCV_1x_ProG_B	292	15	6
Nor.34_1X_PCV_1x_ProG_B	294	14	7
P69_5.2.20_CovA.2H	294	2	2
P78_5.9.20_CovA.2A	294	56	52
P43_5.1.20_CovA.2B	294	42	37
P43_5.7.20_CovA.2C	294	31	27
P52_4.30.20_CovA.2D	294	16	16
P52_5.7.20_CovA.2E	294	16	14
P1_1X_PCV_1x_ProG_B	55	5	4
P2_1X_PCV_1x_ProG_B	55	0	0
P3_1X_PCV_1x_ProG_B	55	12	9
P4_1X_PCV_1x_ProG_B	55	8	7
HD6_1X_PCV_1x_ProG_B	55	6	5
HD7_1X_PCV_1x_ProG_A	75	21	16
HD8_1X_PCV_1x_ProG_A	75	9	3
HD9_1X_PCV_1x_ProG_A	75	12	9
BB3_1X_PCV_1x_ProG_A	75	6	3
V3_1X_PCV_1x_ProG_A	75	9	9
V5_1X_PCV_1x_ProG_A	75	15	9
V6_1X_PCV_1x_ProG_A	75	1	0
P5_1X_PCV_1x_ProG_A	75	1	0
HD7_1X_PCV_1x_ProG_B	56	9	5
HD8_1X_PCV_1x_ProG_B	56	9	6
HD9_1X_PCV_1x_ProG_B	55	9	4
BB3_1X_PCV_1x_ProG_B	55	6	4
V3_1X_PCV_1x_ProG_B	56	1	1
V5_1X_PCV_1x_ProG_B	56	11	8
V6_1X_PCV_1x_ProG_B	56	5	4
P5_1X_PCV_1x_ProG_B	56	2	2
BB4_1X_PCV_1x_ProG_A	73	12	6
BB5_1X_PCV_1x_ProG_A	73	20	12
BB6_1X_PCV_1x_ProG_A	73	13	7
BB7_1X_PCV_1x_ProG_A	73	3	0
BB4_1X_PCV_1x_ProG_B	59	5	3
BB5_1X_PCV_1x_ProG_B	60	11	7
BB6_1X_PCV_1x_ProG_B	59	10	8
BB7_1X_PCV_1x_ProG_B	59	0	0
Sblk_1X_PCV_1X_ProG_F	60	15	15
JA.s_1X_PCV_1X_ProG_F	60	8	7
P77_5.4.20_CovA.3F	228	14	11
P77_5.11.20_CovA.3G	228	3	3
P88_5.5.20_CovA.3H	228	3	2
P69_5.9.20_CovA.3A	228	30	25
P81_5.2.20_CovA.3B	225	24	22
P81_5.9.20_CovA.3C	228	15	13
P51_5.3.20_CovA.3D	228	6	5
P51_5.10.20_CovA.3E	226	4	3
P87_5.6.20_CovA.4F	218	7	7
P87_5.13.20_CovA.4G	217	2	2
P73_5.4.20_CovA.4H	218	1	1
P88_5.12.20_CovA.4A	219	34	31
P90_5.6.20_CovA.4B	216	26	23
P90_5.13.20_CovA.4C	216	16	15
P48_5.3.20_CovA.4D	216	13	11
P48_5.10.20_CovA.4E	217	13	10
P61_5.14.20_CovA.5F	207	15	15
P44_5.9.20_CovA.5G	207	1	0
Nor.35_1X_PCV_1x_ProG_A	207	9	8
Nor.36_1X_PCV_1x_ProG_A	207	21	13
Nor.37_1X_PCV_1x_ProG_A	208	12	8
Nor.38_1X_PCV_1x_ProG_A	208	11	10
Nor.39_1X_PCV_1x_ProG_A	208	7	6
Nor.40_1X_PCV_1x_ProG_A	208	19	13
Nor.41_1X_PCV_1x_ProG_A	207	12	12
Nor.42_1X_PCV_1x_ProG_A	208	12	8
P44_Unk_CovA.5H	208	1	1
P73_5.11.20_CovA.5A	208	24	22
P82_5.13.20_CovA.5B	208	26	24
P47_5.8.20_CovA.5C	208	11	11
P47_5.15.20_CovA.5D	208	11	11
P61_5.7.20_CovA.5E	208	8	7
P83_5.9.20_CovA.6F	270	11	11
P83_5.16.20_CovA.6G	270	3	2
Nor.35_1X_PCV_1x_ProG_B	270	21	16
Nor.36_1X_PCV_1x_ProG_B	270	18	15
Nor.37_1X_PCV_1x_ProG_B	270	14	13
Nor.38_1X_PCV_1x_ProG_B	269	15	12
Nor.39_1X_PCV_1x_ProG_B	270	7	6
Nor.40_1X_PCV_1x_ProG_B	270	16	12
Nor.41_1X_PCV_1x_ProG_B	269	14	12
Nor.42_1X_PCV_1x_ProG_B	269	22	13
P55_5.9.20_CovA.6H	270	4	4
P57_5.13.20_CovA.6A	269	41	36
P46_5.9.20_CovA.6B	270	28	24
P46_5.17.20_CovA.6C	269	12	10
P64_5.8.20_CovA.6D	269	11	10
P64_5.15.20_CovA.6E	269	22	20
P53_5.16.20_CovA.7F	221	14	12
P86_5.14.20_CovA.7G	221	2	2
P86_5.20.20_CovA.7H	221	1	1
P55_5.16.20_CovA.7A	221	18	15
P65_5.11.20_CovA.7B	220	16	15
P65_5.18.20_CovA.7C	220	7	7
P59_Unk_CovA.7D	220	7	6
P53_5.9.20_CovA.7E	220	14	14
P93_5.29.20_CovA.8F	223	19	17
P91_5.21.20_CovA.8G	224	7	7
P91_5.28.20_CovA.8H	224	6	6
P63_5.16.20_CovA.8A	225	15	14
P63_5.23.20_CovA.8B	224	10	8
P84_5.15.20_CovA.8C	225	19	17
P84_5.22.20_CovA.8D	225	14	14
P93_5.19.20_CovA.8E	225	11	11
Superblock_PCV_B	200	1	0
JA_Serum_PCV_A	200	0	0
BB.8_1X_PCV_1x_ProG_A	200	17	13
BB.9_1X_PCV_1x_ProG_A	199	17	16
BB.10_1X_PCV_1x_ProG_A	200	7	4
BB.11_1X_PCV_1x_ProG_A	200	11	9
BB.12_1X_PCV_1x_ProG_A	199	10	7
BB.13_1X_PCV_1x_ProG_A	200	13	10
BB.14_1X_PCV_1x_ProG_A	200	11	5
Sblk_1X_PCV_1X_ProG_E	200	44	41
JA_Serum_PCV_B	200	1	0
P74_5.11.20_CovA.9A	200	24	22
P74_5.18.20_CovA.9B	200	15	14
P45_5.16.20_CovA.9C	200	12	10
P45_5.25.20_CovA.9D	197	8	7
Superblock_PCV_A	198	2	0

